docs/source/cli-reference/synopsis.rst

```diff
@@ -18,28 +18,30 @@ Toolkit Command Line Interface in the following sections:
 Installation
 ------------
 
-Scancode-Toolkit installation can be installed from ``pip``, the default Python Package Manager.
-However, there are more ways to perform an installation, and refer the following sections for
+Scancode-Toolkit installation can be done by downloading ScanCode as an application, which
+is recommended generally. For users familiar with ``pip``, ScanCode is also available and can be
```

Comment:
Replace "For users familiar with pip" with "For users who wish to use ScanCode as a library, it can be installed via `pip`"